Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,3 @@
+# rkt stop
+
+Given a list of pod UUIDs, rkt stop will shut them down cleanly.
```

Comment:
You can add copy/paste of the command output, to see how it looks like.

With and without `--force`.